wpilib/tests/test_driverstation.py

```diff
@@ -109,8 +109,6 @@ def test_getStickAxis_limits(dsmock, halmock):
         dsmock.getStickAxis(-1, 1)
     with pytest.raises(IndexError):
         dsmock.getStickAxis(dsmock.kJoystickPorts, 1)
-    with pytest.raises(IndexError):
-        dsmock.getStickAxis(0, -1)
```

Comment:
not sure why upstream removed the check

Reply:
Hm. We should ask whether that was intentional. I suspect not.

Reply:
Upstream is incorrect, we should make ours correct.